alica_engine/include/engine/AlicaContext.h

```diff
@@ -608,7 +664,12 @@ bool AlicaContext::setOptions(const std::vector<std::pair<std::string, T>>& keyV
             oldKeyValuePairs.push_back(oldKeyValuePair);
         }
     } catch (const YAML::Exception& e) {
-        ALICA_WARNING_MSG("AC: Could not set config values: " << e.msg);
+        if (!_logger) {
+            std::cerr << "AC: Could not set config values: " << e.msg << std::endl;
```

Comment:
Can we not use the default logger for this?

Reply:
We can create a local temporary instance of a default logger to log these msgs, but we cant initialize the default logger with setLogger, because that sets the DefaultLogger as a singleton.